lib/spell-check-rules.js

```diff
@@ -12,5 +12,21 @@ module.exports = [
 	{
 		test: /\bjavascript\b/gi,
 		value: 'JavaScript'
+	},
+	{
+		test: /\bmac\s?OS\b/gi,
+		value: 'macOS'
+	},
```

Comment:
Can you also add one that corrects `OS X` and `OSX` and `Mac OSX` to `macOS`?

Reply:
Done